src/nvector/parhyp/nvector_parhyp.c

```diff
@@ -442,9 +442,9 @@ void N_VSetArrayPointer_ParHyp(sunrealtype *v_data, N_Vector v)
 }
 
 
-void *N_VGetCommunicator_ParHyp(N_Vector v)
+MPI_Comm N_VGetCommunicator_ParHyp(N_Vector v)
```

Comment:
It is my impression that hypre can be built without MPI, so should this be `SUNComm` instead, or do we require that we only link against MPI-enabled hypre installations?